src/main/io/gps.c

```diff
@@ -1955,7 +1955,7 @@ void onGpsNewData(void)
 
     lastTimeUs = timeUs;
 
-    if (!(STATE(GPS_FIX) && gpsSol.numSat >= GPS_MIN_SAT_COUNT)) {
+    if (!(STATE(GPS_FIX) || gpsSol.numSat >= GPS_MIN_SAT_COUNT)) {
```

Comment:
// not (A or B) = (not A) and (not B)
// not (A and B) = (not A) or (not B)

Reply:
Good old DeMorgan… https://en.m.wikipedia.org/wiki/De_Morgan%27s_laws

Reply:
Hmm lets think about this.
The intent is, while flying home, to not interrupt the rescue unless the GPS data is such that the quad would likely be going completely haywire.
So long as the GPS reports a 3D fix, regardless of the satellite count, it is probably sufficient to keep flying home.
However this would not be a sufficient requirement for determining a valid home point.
So we would need to look at this in relation to the rescue process itself.
The test for arming in GPS Rescue mode, if a valid home point is required, should be the presence of both a 3D fix and at least min satellites.
The test for continuing the rescue should be only that we have a 3D fix.

Reply:
Agree, we should remove the second part of the condition here

Reply:
Yes I think we only require the 3D fix check here.